src/yb/client/tablet_server.h

```diff
@@ -83,5 +83,16 @@ struct YBTabletServerPlacementInfo {
   }
 };
 
+struct YBActiveUniverseHistoryInfo {
+  std::string wait_state;
+
+  template <class PB>
+  static YBActiveUniverseHistoryInfo FromPB(const PB& pb) {
+    return YBActiveUniverseHistoryInfo {
+      .wait_state = pb,
```

Comment:
Generally, FromPB/ToPB is used for searializing/desearilzing from PB to a class and vice-versa.

wait_state is a string, how does it get assigned to a PB?

Reply:
Right now, the wait_state in RpcCallInProgressPB is just a string, so the PB (in FromPB) is just a std::string. But I think this will change in the future and we will have something like

`WaitStatePB wait_state = 8`

In that case, this FromPB can be modified to deserialize WaitStatePB in the future.